README.md

```diff
@@ -68,12 +70,12 @@ Or in a component
 </template>
 <script lang="ts">
 import { Options, Vue } from 'vue-class-component';
+import { injectionToken } from 'vue-auth0-plugin'
 @Options({
   inject: ['auth'],
```

Comment:
Didn't you miss using the `injectionToken` here?

Reply:
good catch. Thanks!